web_src/css/repo.css

```diff
@@ -1142,12 +1142,19 @@
 .repository.view.issue .comment-list .event .detail {
   margin-top: 4px;
   margin-left: 14px;
+  display: flex;
+  align-items: center;
+  gap: 0.25rem;
 }
 
 .repository.view.issue .comment-list .event .detail .svg {
   margin-right: 2px;
 }
 
+.repository.view.issue .comment-list .event .detail .muted-links {
+  display: flex;
+}
+
```

Comment:
Arguably a `gt-df` would be better here than this monstrosity of a selector.

Reply:
Done.